ocaml/utils/warnings.ml

```diff
@@ -452,6 +454,10 @@ let descriptions = [
     names = ["tmc-breaks-tailcall"];
     description = "A tail call is turned into a non-tail call \
                    by the @tail_mod_cons transformation." };
+  { number = 73;
+    names = ["total-match-in-pattern-guard"];
+    description = "Cases in pattern guard are total. can be refactored into nested match."
```

Comment:
```suggestion
    description = "Cases in pattern guard are exhaustive, and so the pattern guard can instead be expressed as a case whose body is an exhaustive match."
```

Reply:
I think using "exhaustive" in user-facing documentation is better than "total", for consistency. (Also, I'm suggesting a clarification as to what exactly can be refactored into a nested match.)

Reply:
gotcha, this makes sense. I think I erroneously committed some partial description here.